sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -75,6 +79,27 @@
     private PermissionManager permissionManager;
     */
 
+	CompletionListener subManagerListener = new CompletionListener() {
+		boolean allSucceeded = true;
```

Comment:
If a single sub manager fails do we even continue? Or should the SdlManager shutdown and throw an error?